.travis.yml

```diff
@@ -1,20 +0,0 @@
-notifications:
-  email: false
-sudo: false
-language: python
-cache:
-  - pip
-python:
-  - '3.6'
-  - '3.7'
-
-before_install:
-  - travis_retry pip install --upgrade pip setuptools py
-  - travis_retry curl -sSL https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py | python - --version 0.12.16;
-    source $HOME/.poetry/env;
-
-install:
-  - travis_retry poetry install
-
-script:
-  - "./run-tests.sh"
```

Comment:
so you remove travis, but you're not replacing it with drone here, are you? don't you want to do that simultaneously?